Description: switch to using gnome-session
 temporary patch pending budgie-session upload.  This patch
 switches to using gnome-session
Author: David Mohammed <fossfreedom@ubuntu.com>
Forwarded: not-needed
Reviewed-By: David Mohammed <fossfreedom@ubuntu.com>
Last-Update: 2024-01-28

--- budgie-desktop-10.9.orig/src/session/budgie-desktop.in
+++ budgie-desktop-10.9/src/session/budgie-desktop.in
@@ -13,4 +13,4 @@ if [ -z $XDG_CURRENT_DESKTOP ]; then
   export XDG_CURRENT_DESKTOP
 fi
 
-exec budgie-session --builtin --session=org.buddiesofbudgie.BudgieDesktop $*
+exec gnome-session --builtin --session=org.buddiesofbudgie.BudgieDesktop $*
